backend/server/routes/index.js

```diff
@@ -7,9 +7,13 @@ const router = Router()
 /*
 USERS
 */
-router.route('/users').post(user.addUser)
+router.route('/users')
+    .post(user.addUser)
+    .put(user.modifyUserByID)
+    .delete(user.deleteUserByID)
```

Comment:
Since we're not using ID anymore, we probably should name the functions modifyUser and deleteUser instead

Reply:
Got it